python/ray/_private/ray_logging/logging_config.py

```diff
@@ -35,24 +109,8 @@ def f():
             2024-06-03 07:53:50,815 INFO test.py:11 -- This is a Ray task job_id=01000000 worker_id=0dbbbd0f17d5343bbeee8228fa5ff675fe442445a1bc06ec899120a8 node_id=577706f1040ea8ebd76f7cf5a32338d79fe442e01455b9e7110cddfc task_id=c8ef45ccd0112571ffffffffffffffffffffffff01000000
 
     Args:
-        encoding: Encoding type for the logs. The valid value is 'TEXT'
+        encoding: Encoding type for the logs. The valid values are
+            {list(_dict_config_provider.get_supported_encodings())}
```

Comment:
is this why the doc needs to be rendered at run time

Reply:
Yes.